test/powershell/Language/Scripting/NativeExecution/NativeUnixGlobbing.Tests.ps1

```diff
@@ -38,6 +38,12 @@ Describe 'Native UNIX globbing tests' -tags "CI" {
     It 'Should return the original pattern if there are no matches' {
         /bin/echo $TESTDRIVE/*.nosuchfile | Should Match "\*\.nosuchfile$"
     }
+    It 'Should not expand double-quoted strings' {
+        /bin/echo "*" | Should BeExactly "*"
+    }
+    It 'Should not expand single-quoted strings' {
+        /bin/echo '*' | Should BeExactly "*"
+    }
```

Comment:
This change only impacts expansion of wildcard characters. ':' isn't a wildcard character so I don't see a reason to add this test.